content/docs/en/routing/vue-router.md

```diff
@@ -1,43 +1,41 @@
 ---
 title: Vue Router
-contributors: [eddyverbruggen, rigor789]
+contributors: [eddyverbruggen, rigor789, ikoevska]
 ---
 
-If [Manual Routing](/en/docs/routing/manual-routing) doesn't cut it for your use-case,
-then you'll be happy to learn [the Vue router](https://router.vuejs.org/en/) is supported.
+> Currently, integration with Vue Router is **experimental**. If you want to use a non-experimental approach, you can try [manual routing](/en/docs/routing/manual-routing).  
 
-With the router, there are two types of routing that you can use. Component based routing, and page based routing.
+With the router, you can choose between [component-based routing](https://router.vuejs.org/guide/essentials/named-views.html) and page-based routing. In a mobile app, you are more likely to implement page-based routing.
```

Comment:
I'm not sure if directing users to **named views** is right, it should just be normal `<router-view>` (which is included in the linked page, but could be confusing at first)

Maybe https://router.vuejs.org/guide/ would be a better link - although it includes mostly web related information which is also not ideal. :/

Reply:
I'm switching the link to https://router.vuejs.org/guide/essentials/named-views.html#nested-named-views Would that make more sense?

Reply:
Just found https://router.vuejs.org/api/#router-view maybe that would be the best here?

Reply:
Yep, seems to work better. Changed :)